### PR TITLE
Update vendor for amp-ui

### PR DESCRIPTION
**Pull request by freignat91**
Add following packages for amp-ui

- github.com/gorilla/mux
- github.com/gorilla/context
- github.com/phyber/negroni-gzip/gzip
- github.com/urfave/negroni

## test 

- ampmake cleanall build

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -80,17 +80,25 @@ import:
 - package: gopkg.in/olivere/elastic.v5
   version: ^5.0.35
 - package: gopkg.in/yaml.v2
-## Required to fix package resolution from github.com/ory-am/ladon
```
Comment (bquenin) [inline glide.yaml]: Please restore comments

File: glide.yaml [hunk 2]
```diff
@@ -80,17 +80,25 @@ import:
...
 - package: github.com/pkg/errors
   version: ^0.8.0
-## Required to fix package resolution from github.com/docker/docker
```
Comment (bquenin) [inline glide.yaml]: Please restore comments

File: glide.yaml [hunk 3]
```diff
@@ -80,17 +80,25 @@ import:
...
-- package: github.com/gogo/protobuf/types
-  version: 0.4
-## Required to fix package resolution from github.com/coreos/etcd, should be located in testImport
```
Comment (bquenin) [inline glide.yaml]: Please restore comments

**Review by bquenin — CHANGES_REQUESTED** [review]
LGTM

File: glide.yaml (hunk 2 above)
Comment (freignat91) [reply glide.yaml]: ok

File: glide.yaml (hunk 3 above)
Comment (freignat91) [reply glide.yaml]: ok

File: glide.yaml (hunk 1 above)
Comment (freignat91) [reply glide.yaml]: ok

**Review by bquenin — APPROVED** [review]

**Review by subfuzion — APPROVED** [review]
👍

**Pull request merged**